javascript/ql/src/Declarations/MissingThisQualifier.ql

```diff
@@ -50,5 +50,13 @@ where maybeMissingThis(call, intendedTarget, gv)
           decl.isNamespaceExport() and
           call.getContainer().getEnclosingContainer*() instanceof NamespaceDeclaration
         )
+        or
+        // call to global function with additional arguments
+        exists (Function self |
+          intendedTarget.getBody() = self and
+          call.getEnclosingFunction() = self and
+          call.flow().(DataFlow::CallNode).getNumArgument() > self.getNumParameter() and
```

Comment:
Remind me, how does `getNumParameter()` behave in the presence of rest parameters?